### ISSUE TITLE
HitBTC v2 -- ccxt.base.errors.ExchangeError: hitbtc2 does not have market symbol None

### ISSUE THREAD

**Issue by nmiculinic** [ccxt/ccxt#609]
Can you make HitBTC2 on market symbol None to fetch all symbol pairs? 

Concrete function is ```fetch_closed_orders```

This is kinda the API request:
```python
            trade_history = await exchange.private_get_history_trades({
                'by': 'ts',
                'start_index': 0,
                'max_results': 1000,
                'from': str(arrow.get(start)),
            })
```

**Comment by kroitor** [comment]
@nmiculinic sure! Will look into it and resolve this asap.

**Comment by kroitor** [comment]
This is fixed in 1.10.148. Also, a few other improvements to HitBTC v2 were added to match their new API. Thx for reporting!

References: #580
